src/py/flwr/server/strategy/__init__.py

```diff
@@ -16,7 +16,10 @@
 
 
 from .bulyan import Bulyan as Bulyan
-from .dp_adaptive_clipping import DifferentialPrivacyClientSideAdaptiveClipping
+from .dp_adaptive_clipping import (
+    DifferentialPrivacyClientSideAdaptiveClipping,
+    DifferentialPrivacyServerSideAdaptiveClipping,
+)
 from .dp_fixed_clipping import (
     DifferentialPrivacyClientSideFixedClipping,
     DifferentialPrivacyServerSideFixedClipping,
```

Comment:
I think it would be better if we reformat them as `from xx import yyy as yyy`, in case that we want to change the class names after a stable release. Wdyt?